From d7c7a71a9cfb790fa22d542d3ec82c873d266721 Mon Sep 17 00:00:00 2001
From: Artem Senichev <a.senichev@yadro.com>
Date: Fri, 5 Apr 2019 13:39:37 +0300
Subject: [PATCH] Setup APSS channels (power consumption)

- Set gain and channel offset values for sensors;
- Renumbered IPMI sensors IDS to make the compact sequence.

Output values of power consumption sensors now differ from the PDU output
power by 5% (~20W in idle CPU state), this difference depends on some
components not connected to APSS (e.g. fans).

Signed-off-by: Artem Senichev <a.senichev@yadro.com>
---
 vesnin.xml | 60 +++++++++++++++++++++++++++---------------------------
 1 file changed, 30 insertions(+), 30 deletions(-)

diff --git a/vesnin.xml b/vesnin.xml
index 7eef822..7f990b5 100644
--- a/vesnin.xml
+++ b/vesnin.xml
@@ -1357,7 +1357,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xB0</value>
+			<value>0xAB</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1401,7 +1401,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA4</value>
+			<value>0xAC</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1412,7 +1412,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA5</value>
+			<value>0xAA</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1489,7 +1489,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xAC</value>
+			<value>0xA5</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1500,7 +1500,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xAD</value>
+			<value>0xA6</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1511,7 +1511,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xAA</value>
+			<value>0xA7</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1522,7 +1522,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xAB</value>
+			<value>0xA8</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1533,7 +1533,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA6</value>
+			<value>0xA9</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1577,7 +1577,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA2</value>
+			<value>0xA1</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1588,7 +1588,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA3</value>
+			<value>0xA2</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1599,7 +1599,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA1</value>
+			<value>0xA3</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -1610,7 +1610,7 @@
 		</property>
 		<property>
 			<id>IPMI_SENSOR_ID</id>
-			<value>0xA7</value>
+			<value>0xA4</value>
 		</property>
 	</globalSetting>
 	<globalSetting>
@@ -23846,7 +23846,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>11600</default>
+			<default>44929</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -23858,7 +23858,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>48</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
@@ -23925,7 +23925,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>11600</default>
+			<default>44929</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -23937,7 +23937,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>-10</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
@@ -24004,7 +24004,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>34500</default>
+			<default>44929</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24083,7 +24083,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>34500</default>
+			<default>44929</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24162,7 +24162,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>22700</default>
+			<default>33697</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24174,7 +24174,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>54</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
@@ -24241,7 +24241,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>22600</default>
+			<default>33697</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24253,7 +24253,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>-16</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
@@ -24320,7 +24320,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>34500</default>
+			<default>33697</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24399,7 +24399,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>34500</default>
+			<default>33697</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24478,7 +24478,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>11000</default>
+			<default>6739</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24490,7 +24490,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>32</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
@@ -24794,7 +24794,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>34500</default>
+			<default>6739</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -24806,7 +24806,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>26</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
@@ -25742,7 +25742,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GAIN</id>
-			<default>6495</default>
+			<default>6500</default>
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_GROUND</id>
@@ -25754,7 +25754,7 @@
 		</attribute>
 		<attribute>
 			<id>ADC_CHANNEL_OFFSET</id>
-			<default>10</default>
+			<default>0</default>
 		</attribute>
 		<attribute>
 			<id>BUS_TYPE</id>
-- 
2.21.0

